app/controllers/users_controller.rb

```diff
@@ -18,18 +23,54 @@ def create
       flash[:success] = t "welcome"
       redirect_to @user
     else
-      flash[:danger] = t ".sign_up_fail"
+      flash.now[:danger] = t ".sign_up_fail"
       render :new
     end
   end
 
-  def update; end
+  def update
+    if @user.update(user_params)
+      flash[:success] = t ".update_success"
+      redirect_to @user
+    else
+      flash.now[:danger] = t ".update_fail"
+      render :edit
+    end
+  end
 
-  def destroy; end
+  def destroy
+    @user.destroy
+    if @user.destroyed?
```

Comment:
```suggestion
    if @user.destroy
```
Như này thôi cũng được mà em hàm destroy trả về true false mmaf